src/Materials/PBR/pbrBaseMaterial.ts

```diff
@@ -363,26 +363,30 @@ export abstract class PBRBaseMaterial extends PushMaterial {
     /**
      * Intensity of the direct lights e.g. the four lights available in your scene.
      * This impacts both the direct diffuse and specular highlights.
+     * @hidden
      */
-    protected _directIntensity: number = 1.0;
+    public _directIntensity: number = 1.0;

```

Comment:
I don't know about this. We should check with Seb but he is out on vacation for a while. @deltakosh or @RaananW Do you have an opinion on this?

Reply:
I asked seb before he left. He suggested this as our simplest way to expose this functionality.